apps/test/applabTest.js

```diff
@@ -174,7 +174,7 @@ describe('getText/setText commands', function () {
 
       it('does add leading newline for leading empty div', function () {
         element.innerHTML = '<div><br></div><div>text</div><div>with</div><div>leading empty div</div>';
-        assert.equal(getInnerText(element), '\n\ntext\nwith\nleading empty div');
+        assert.equal(getInnerText(element), '\ntext\nwith\nleading empty div');
```

Comment:
I think this was an actual bug in the existing tests - if you expand and look at the test above this one it checks 'no leading newline' expecting `text\nwith...` - so I'm not sure why this 'with leading newline' test expected two leading newlines `\n\ntext\nwith...`